problem_builder/mentoring.py

```diff
@@ -1101,9 +1115,7 @@ def try_again(self, data, suffix=''):
         }
 
     def author_preview_view(self, context):
-        context = context.copy() if context else {}
-        context['author_preview_view'] = True
-        return super(MentoringWithExplicitStepsBlock, self).author_preview_view(context)
+        return self.student_view(context)
```

Comment:
Yes, but if it renders `student_view` as a fallback, Studio will wrap the XBlock in a bunch of HTML (header, edit controls) that we don't want. Studio looks at the name of the view that's being rendered when deciding to wrap the element or not.